docs/Bundler/integration.mdx

```diff
@@ -59,6 +59,20 @@ const bundler: IBundler = new Bundler({
 });
 ```
 
+## New Integration Example
+If you do not wish to use any custom bundler you can simply pass the bunlder url in the smart account config.
```

Comment:
similarly here, this is a config change and only needs to be added in the create method config and account integration